packages/contractkit/src/kit.ts

```diff
@@ -67,6 +73,38 @@ export class ContractKit {
     return toTxResult(promiEvent)
   }
 
+  async sendSignedTransaction(tx: Tx): Promise<TransactionResult> {
```

Comment:
i don't like this.
IMO, tx signing should be transparent, not something that you "say explicitly". You are already saying that when you use `from` with a local account

Reply:
> You are already saying that when you use from with a local account

Not exactly true, as of now, someone might still want to have an account in both web3 as well as a locally running geth node and would want to sign to using the geth node. I can merge these two into one but I am sure someone later is going to demand the other use-case.

Reply:
why would something want like.. I think that's kind of a contrived example.

One should be able to do:

```ts
const stableToken = kit.contract.getStableToken()

const tx = await stableToken.transfer(value, to).send()

```

and that `send()` use a local account or not, depending on the `from`. which in this example is the default, but you can also do `.send({from: someAccount})`

Reply:
I removed `sendSignedTransaction` completely. It should not be required at all (as confirmed by the unittest ts-signing.test.ts as well)